src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -121,4 +124,22 @@ public static Set<Tag> parseTags(Collection<String> tags) throws ParseException
         }
         return tagSet;
     }
+
+    /**
+     * Parses a {@code String role} into a {@code Role}.
+     */
+    public static Role parseRole(String role) throws ParseException {
+        requireNonNull(role);
+        RoleHandler rh = new RoleHandler();
+        String trimmedRole = role.trim();
```

Comment:
This might have to consider cases where multiple roles are in a single string? argumentMultiMap where this would be called passes a List with a single string rather than each individual role string separated

Reply:
I see, will implement parseRoles for multiple roles provided as argumentMultiMap!